core/common/assets/js/api/modules/command-data.js

```diff
@@ -177,4 +178,21 @@ export default class CommandData extends CommandBase {
 	applyAfterUpdate( data, args = {} ) {// eslint-disable-line no-unused-vars
 		return data;
 	}
+
+	/**
+	 * Called after apply() failed.
+	 *
+	 * @param e
+	 */
+	onCatchApply( e ) {
+		const status = e?.data?.status || 0,
```

Comment:
In what situation there isn't `e`?

Reply:
The problem here is that "e" not always has "data" and "status".
in the current time, we do not have a consistent structure of errors that returns from the server, in the future when we will fix this issue, it can be removed (I will add a TODO).